.github/workflows/test.yaml

```diff
@@ -0,0 +1,34 @@
+name: Test
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+permissions:
```

Comment:
```permissions: {}``` would work here instead of listing them all as none

Reply:
Good catch, thanks Ryan !